wgsl/index.bs

```diff
@@ -6835,31 +6835,36 @@ That's not a full user-defined function declaration.
 
 <table class='data'>
   <thead>
-    <tr><th>Precondition<td>Conclusion<td>Notes
+    <tr><th>Precondition<th style="width:45%">Conclusion<td>Description
   </thead>
   <tr algorithm="vector all">
-    <td>|e|: vecN&lt;bool&gt;
-    <td>`all(`|e|`)`: bool
+    <td>|T| is bool
+    <td>`all(`|e|`:` vecN<|T|`> ) -> ` |T|
```

Comment:
You've spotted that the spec is inconsistent in how it describe builtins.

Sometimes, like this case, it uses type assertion, as described in https://gpuweb.github.io/gpuweb/wgsl/#type-checking-section:

          all(e) : bool


Then sometimes it writes function prototypes, like was done for select (even in the same table)!

The spec doesn't even define what a function prototype is, so I personally lean to fixing everything by going in the other direction, ie. by using type assertions.

Reply:
Please can we make a decision on how we want this formatted, because changing everything will affect all the CTS test-id codes.
I don't mind which way we go, but fixing this either way sooner rather than later would save a lot of churn.